src/main/resources/openapi/PrintAPIs.yaml

```diff
@@ -786,10 +786,20 @@ components:
         - dateTime
 
     SupportingInformationFormat:
+      title: SupportingInformationFormat
```

Comment:
Can't extend this as VCA and other flows uses this, so had to add a new format enum

Reply:
Should the temporary certificate not also support the other supporting information formats?

Reply:
Not sure I understand the question? @vishalgupta-valtech's change doesn't affect temp certs.

Reply:
@Neil-Massey I don't see temporary certs using `supportingInformationFormat` in spec before and also they are different to AEDs as AEDs needs to be generated (if requested) after actual removal from vca.

<img width="524" alt="image" src="https://user-images.githubusercontent.com/73763010/229109011-508b8634-53d9-461e-8595-736ddd711b46.png">

Reply:
Sorry, missed two things, 1 this is a bug fix and 2 the endpoint I was thinking doesn't take a supporting document format

```
/eros/{eroId}/temporary-certificates/{gssCode}/explainer-document
```